packages/common/src/utils/challenges.ts

```diff
@@ -307,9 +307,12 @@ export const challengeRewardsConfig: Record<
     id: 'trending-underground'
   },
   o: {
-    title: 'Airdrop - Februrary 2025',
-    description: () => 'Claim your $AUDIO before it expires!',
-    fullDescription: () => 'Claim your $AUDIO before it expires!',
+    title: 'Airdrop 2: Artist Appreciation',
+    description: () =>
+      `We're thrilled to reward our talented artist community for driving Audius' growth and success! \n\n Claim your tokens before they expire on 05/13/25!`,
```

Comment:
shouldn't short description be 
Claim your $AUDIO airdrop before it expires!

we still want the short one to show up on the tile (not the modal) right?